### ISSUE TITLE
Good First Issue | Write tests for data_parser.py

### ISSUE THREAD

**Issue by samgdotson** [arfc/pride#53]
The functions in ``temoa-uiuc/data_parser.py`` need to have associated unit tests.

**This is a good first issue for new students/researchers**. 

For guidance on writing unit tests, please see "Effective Computation in Physics by Huff, Scopatz."

This issue can be closed when:
- [ ] ``test_data_parser.py`` has been added to the ``tests`` folder in ``pride/`` (this folder may need to be created, depending on the status of Issue #103). 
- [ ] Each function in ``data_parser.py`` has an appropriate unit test, covering edge and corner cases if applicable.
- [ ] All tests pass by running ``pytest`` in the top level of the repository.

**Comment by samgdotson** [comment]
Adding @npanczyk after expressing interest during morning meeting.

**Comment by katyhuff** [comment]
@samgdotson @yardasol @npanczyk 

What is the status of this?
Unless there's a good reason to assign two people, it is best if only one person is assigned to each issue. Otherwise, there is a bystander effect.

**Comment by samgdotson** [comment]
@npanczyk was working on it but has sidelined this to work on the "fountain reactor" thing.

**Comment by npanczyk** [comment]
Yes, I was in the midst of this (mostly held back by trying to get temoa to run on my computer) but paused to work on fountain-2021. Happy to go back to it once we finish the lit review, but I was under the impression that this was less of a priority? Let me know if that's still the case.

**Comment by yardasol** [comment]
> Yes, I was in the midst of this (mostly held back by trying to get temoa to run on my computer) but paused to work on fountain-2021. 

@npanczyk I was also having issues with getting TEMOA to run on my machine! I'd be happy to walk through the set up with you if you think it would be helpful.

**Comment by npanczyk** [comment]
@yardasol Yes! That would be great, maybe in a couple weeks we can go through the set up so that I can devote all of my attention to writing the tests thereafter. I left off with some sort of latex error, I believe, so we'll probably have to start from there.

**Comment by katyhuff** [comment]
Totally fine to put it on pause, I just like to keep track of the status of these things.
